proto/ibc/lightclients/tendermint/v1/tendermint.proto

```diff
@@ -57,6 +57,7 @@ message ClientState {
   // allow_update_after_misbehaviour is deprecated
   bool allow_update_after_misbehaviour = 11
       [deprecated = true, (gogoproto.moretags) = "yaml:\"allow_update_after_misbehaviour\""];
+  bytes absence_sentinel_value = 12 [(gogoproto.moretags) = "yaml:\"absence_sentinel_value\""];
```

Comment:
Sorry if if this is a silly question, but I was just wondering why this sentinel value would be needed for Tendermint, since Tendermint supports absence proofs, right? I would have expected this sentinel value to be added to the `ClientState` proto of light clients that do not support absence proofs instead... Or do you envision a future where there would be a version of Tendermint using a different state tree that doesn't support absence proofs?

Or is this sentinel value that should be supported by all light clients anyway?

Reply:
I see, thanks for the explanation!